sdc/tests/test_dataframe.py

```diff
@@ -891,7 +891,7 @@ def test_impl(n):
 
     def test_df_fillna1(self):
         def test_impl(df):
-            return df.fillna(5.0)
+            return df.fillna(0.)
```

Comment:
What is the reason to change it?

Reply:
Its has attention to other PR, I forget delete this